op-node/service.go

```diff
@@ -36,6 +36,10 @@ func NewConfig(ctx *cli.Context, log log.Logger) (*node.Config, error) {
 		return nil, err
 	}
 
+	if !ctx.Bool(flags.BetaRollupLoadProtocolVersions.Name) {
+		rollupConfig.ProtocolVersionsAddress = common.Address{}
```

Comment:
Feels like modifying the rollup config like this warrants a logline, a node runner will probably run into this and not understand why the config isn't being picked up

Reply:
added logging